spec/factories/cryptids.rb

```diff
@@ -2,7 +2,8 @@
   factory :cryptid do
     name { Faker::TvShows::Supernatural.creature }
     range { Faker::Address.state }
-    danger_level { Faker::Number.betwee(from: 1, to: 10) }
-    image { Faker::LoremFlickr.image(seach_terms: ['cryptid']) }
+    danger_level { Faker::Number.between(from: 1, to: 10) }
```

Comment:
So glad you caught this typo I made yesterday, I saw it this morning 😅

Reply:
no biggie quick fix thanks for setting it up with Faker!